charts/kafka-connect/templates/NOTES.txt

```diff
@@ -0,0 +1,9 @@
+CHART NAME: {{ .Chart.Name }}
+CHART VERSION: {{ .Chart.Version }}
+APP VERSION: {{ .Chart.AppVersion }}
+
+** Please be patient while the chart is being deployed **
+
+Check the cluster status by running:
+
+{{ include "kafka-connect.fullname" . }}.{{ .Release.Namespace }}:{{ .Values.service.ports.kafkaConnect }}
```

Comment:
This won't be accessible from outside. Add a command to do `kubectl proxy` as well

Reply:
done